iwyu_ast_util.cc

```diff
@@ -250,6 +253,38 @@ static const RedeclarableTemplateDecl* GetRedeclNotSpecifyingAnyDefArg(
   return nullptr;
 }
 
+template <typename T>
+static void PrintExplicitSpecTplArgs(const T* spec,
+                                     const PrintingPolicy& printing_policy,
+                                     raw_ostream& ostream) {
+  if (spec->isExplicitSpecialization()) {
+    ostream << '<';
+    const TemplateArgumentList& args = spec->getTemplateArgs();
+    const TemplateParameterList* params =
+        spec->getSpecializedTemplate()->getTemplateParameters();
+    for (unsigned i = 0, ie = args.size(); i < ie; ++i) {
+      if (i > 0)
```

Comment:
This looks like a nice utility: https://github.com/llvm/llvm-project/blob/a666d1fd9a1b06e9122531b7e139471f67c1a1d8/llvm/include/llvm/ADT/StringExtras.h#L520

Reply:
Thanks!